src/fluree/sdk/webworker.cljs

```diff
@@ -1,417 +1,385 @@
-  Even data is a javascript object with the following keys:
-  - action - the action to perform
-  - params - parameters for the action we are calling
-  - conn - the connection ID
-  - ref - the reference ID to include in the response
-  "
-  [event]
-  (try
-
-    (let [data       (aget event "data")
-          action-str (aget data "action")
-          conn-id    (aget data "conn")
-          ref        (aget data "ref")
-          params-js  (aget data "params")
-          ;; convert params array into a list, but don't convert the elements to CLJ
-          params     (cond
-                       (or (= action-str "pwGenerate")
-                           (= action-str "transact"))
-                       (obj->clj (first params-js))
-
-                       :else
-                       (js-array->clj-list params-js))]
-
-      (if (conn-closed? conn-id)
-        (when (not= action-str "unregisterQuery")           ;; ignore unregisterQuery actions after connection closed
-          (postMessage {:conn  conn-id
-                        :event action-str
-                        :ref   ref
-                        :data  #js {:status  400
-                                    :message "Connection has been closed."}}))
-        (cond
-          (= action-str "connect")
-          (connect-p (first params) ref)
-
-          (= action-str "close")
-          (close-connection conn-id ref)
-
-          (= action-str "reset")
-          (reset-connection conn-id ref)
-
-          (= action-str "registerQuery")
-          (apply registerQuery conn-id params)
-
-          (= action-str "unregisterQuery")
-          (unregisterQuery conn-id ref)
-
-          (= action-str "transact")
-          (transact conn-id ref params)
-
-
-          :else
-          (throw (ex-info (str "Unknown action: " action-str) {:status 400})))))
-    (catch :default e
-      (let [data       (aget event "data")
-            error-data (if-let [ex-data (ex-data e)]
-                         (merge {:status  (or (:status ex-data) 500)
-                                 :message (ex-message e)}
-                                ex-data)
-                         {:status  500
-                          :message (str "Unexpected error: " (pr-str e))})]
-        (js/console.error "Error executing message:" data)
-        (js/console.error (pr-str error-data))
-        (postMessage {:conn  (aget data "conn")
-                      :event (aget data "action")
-                      :ref   (aget data "ref")
-                      :data  error-data})))))
-
-
-(defn log-error
-  "Log errors in this web worker"
-  [error]
-  (js/console.error error))
-
-(defn init
-  []
-  ;; when we detect that we are in a web worker, register the onmessage handler
-  (when-let [worker? (not (.-document js/self))]
-    (set! (.-onerror js/self) log-error)
-    (set! (.-onmessage js/self) decode-message))
-  ;; let server know we're alive
-  (worker-action nil :connInit))
+(ns fluree.sdk.webworker)
```

Comment:
Are we done with the webworker for good or just temporarily?

Reply:
I think just temporarily. I don't think it was ever updated post v3, so there were many out of data apis and lots of functions that failed the linter because they just returned `:TODO`. I decided to comment out the code instead of totally deleting it so we'd have a reference whenever we get around to re implementing this.